apps/site/next-data/generators/releaseVersions.mjs

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+import nodevuData from './nodevuData.mjs';
+
+/**
+ * This method is used to generate all Node.js versions
+ * for self-consumption during RSC and Static Builds
+ *
+ * @returns {Promise<Array<string>>}
+ */
+const generateAllVersionsData = async () => {
+  const majors = Object.entries(await nodevuData).filter(
```

Comment:
Canm you do the await outside of Object.entries call? Also I believe you should call `nodevuData` as `nodevuPromise` or just `nodeReleasesPromises` it feels weird to call the thing nodevu IDK.